Gemfile.lock

```diff
@@ -2,75 +2,80 @@ PATH
   remote: .
   specs:
     slanger (0.6.0)
-      activesupport (~> 4.2.1)
```

Comment:
Probably easier to bump a minimal number of gems at once and see the build pass or fail